src/librustdoc/html/render/write_shared.rs

```diff
@@ -250,9 +264,14 @@ pub(super) fn write_shared(
     write_toolchain("brush.svg", static_files::BRUSH_SVG)?;
     write_toolchain("wheel.svg", static_files::WHEEL_SVG)?;
     write_toolchain("clipboard.svg", static_files::CLIPBOARD_SVG)?;
-    write_toolchain("down-arrow.svg", static_files::DOWN_ARROW_SVG)?;
-    write_toolchain("toggle-minus.svg", static_files::TOGGLE_MINUS_PNG)?;
-    write_toolchain("toggle-plus.svg", static_files::TOGGLE_PLUS_PNG)?;
+
+    // The following icons are embeded in the CSS file,
+    // unless we're in tweak-the-theme-mode.
+    if !options.enable_minification {
+        write_toolchain("down-arrow.svg", static_files::DOWN_ARROW_SVG)?;
```

Comment:
Just realized but: could you merge the `write_toolchain` call with the `css_icon_url` closure call please? It would make it impossible to forget to update this part.